Lombiq.Hosting.Tenants.Maintenance/Views/Admin/StaggeredMaintenanceDetails.cshtml

```diff
@@ -0,0 +1,167 @@
+          class="ms-1"
+          data-bs-toggle="popover"
+          data-bs-trigger="focus"
+          data-bs-html="true"
+          title="@T["Maintenance Actions Info"]"
+          data-bs-content='
+        <div class="mb-2"><strong>@T["Start new"]:</strong> @T["Start staggered maintenance for all the tenants with a new version number."]</div>
+        <div class="mb-2"><strong>@T["Continue"]:</strong> @T["Continues current maintenance version run."]</div>
+        <div class="mb-2"><strong>@T["Cancel"]:</strong> @T["If there is a running maintenance, you can cancel it."]</div>
+        <div class="mb-2"><strong>@T["Reset current version"]:</strong> @T["Runs the staggered maintenance for all the tenants with the current version again."]</div>
+        <div class="mb-2"><strong>@T["Edit"]:</strong> @T["You can edit all the displayed and functional parameters for the staggered maintenance. You can only save if there is no maintenance running."]</div>
+      '>
+        <i class="fas fa-info-circle text-info" style="cursor: pointer;"></i>
+    </span>
+</div>
+
+<div class="progress mb-3 mt-4">
+    <div
+        id="maintenance-progress-bar"
+        class="
+            progress-bar
+            progress-bar-striped
+            @(part.IsRunning() ? "progress-bar-animated" : string.Empty)
+            @(part.Canceled.Value ? "bg-danger" : string.Empty)
+            @(part.IsFinished() ? "bg-success" : string.Empty)"
+         role="progressbar"
+         aria-valuenow="@currentPercentage"
+         aria-valuemin="0"
+         aria-valuemax="100" style="width: @currentPercentage%">@currentPercentage %</div>
+</div>
+
+<div class="row mb-4">
+    <div class="col-md-6">
+        <dl class="row">
+            <dt class="col-sm-5">@T["Running:"]</dt>
+            <dd class="col-sm-7" data-maintenance-running="@part.IsRunning().ToString().ToLower()">@(part.IsRunning() ? T["Yes"] : T["No"])</dd>
+
+
+            <dt class="col-sm-5">@T["Canceled:"]</dt>
+            <dd class="col-sm-7">@(part.Canceled.Value ? T["Yes"] : T["No"])</dd>
+
+            <dt class="col-sm-5">@T["Started:"]</dt>
+            <dd class="col-sm-7">@part.Started.Value</dd>
+
+            <dt class="col-sm-5">@(part.Canceled.Value ? T["Finished (canceled):"] : T["Finished:"])</dt>
+            <dd class="col-sm-7">@part.Finished.Value</dd>
+        </dl>
+    </div>
+    <div class="col-md-6">
+        <dl class="row">
+            <dt class="col-sm-6">@T["Current version:"]</dt>
+            <dd class="col-sm-6">@part.CurrentVersion.Value</dd>
+
+            <dt class="col-sm-6">@T["Total running tenants:"]</dt>
+            <dd class="col-sm-6">@part.AllTenantCount.Value</dd>
+
+            <dt class="col-sm-6">@T["Processed tenants:"]</dt>
+            <dd class="col-sm-6">@part.ProcessedTenantIds.Count</dd>
+        </dl>
+    </div>
+</div>
+
+<div class="accordion mt-4" id="maintenanceAccordion">
+    <!-- Errors Accordion Item -->
+    <div class="accordion-item">
```

Comment:
Opening an accordion turns its header blue and then it cannot be closed.

Reply:
Not sure why, I'm using the same template as it is on the bootstrap docs. I think it might be related to the version OC uses. I wouldn't spend time on it.

Reply:
If you're using a certain version of a library (wherever it comes from), then you need to write code that's compatible with that version.